oak-segment-aws/pom.xml

```diff
@@ -175,13 +175,13 @@
             <groupId>org.apache.jackrabbit</groupId>
             <artifactId>oak-segment-tar</artifactId>
             <version>${project.version}</version>
-            <scope>provided</scope>
+            <scope>compile</scope>
```

Comment:
@synox, I don't think we need to change the scope of these two dependencies, for both oak-segment-aws and oak-segment-azure modules.